### DISCUSSION TITLE
请问一下，对Qwen进行微调的时候，如何配置验证集，或者如何将训练集的一部分作为验证集？

### DISCUSSION THREAD

**Discussion by purefire** [QwenLM/Qwen#1003] [Q&A]
看到其他模型自带的微调脚本会有一些参数支持带入验证集的路径，或者配置训练集某一个百分比作为验证集，但在Qwen的脚本中似乎没有找到。
另外，是否有办法在训练集中设置一个反例，即将某一个“输入：输出”对规定为不应该出现的输出，防止过拟合？
谢谢。

**Comment by purefire** [comment]
已经解决，看到使用是比较标准的huggingface参数。

**Reply by yihp** [reply]
请问你在哪里看到的相关介绍呢？
